---
layout: archive
permalink: /publications/
author_profile: true
title: "Publications"
---

# Selected Publications

<u>Underline means these authors contribute equally to this work.</u> 

<sup>*</sup>: To whom the correspondence should be addressed.
* <u>Liu, Tianyu</u> and <u>De Brouwer, Edward</u> and Kuo, Tony and Diamant, Nathaniel and Missarova, Alsu and Wang, Hanchen and Hao, Minsheng and Bravo, Hector Corrada and Scalia, Gabriele<sup>* </sup> and Regev, Aviv<sup>* </sup> and Heimberg Graham<sup>* </sup>. Learning multi-cellular representations of single-cell transcriptomics data enables characterization of patient-level disease states. RECOMB, 2025. (oral)
* Liu Tianyu, Xiao Yijia, Luo Xiao, Xu Hua, Zheng Jim Wenjin, Zhao Hongyu<sup>*</sup>. Geneverse: A collection of Open-source Multimodal Large Language Models for Genomic and Proteomic Research. The 2024 Conference on Empirical Methods in Natural Language Processing. ACL, 2024. (poster)
* Liu Tianyu, Chu Tinyi, Luo Xiao, Zhao Hongyu<sup>*</sup>. BAITSAO: Building A Foundation Model for Drug Synergy Analysis Powered by Language Models. Preprint, 2024. (Under review by Nature Biomedical Engineering)
* Liu Tianyu, Lin Yingxin, Luo Xiao, Sun Yizhou, Zhao Hongyu<sup>*</sup>. VISTA Uncovers Missing Gene Expression and Spatial-induced Information for Spatial Transcriptomic Data Analysis. Preprint, 2024. (Under review by Cell Genomics)
* Liu Tianyu, <u>Chen Tianqi</u>, <u>Zheng Wangjie</u>, Luo Xiao, Zhao Hongyu<sup>*</sup>. scELMo: Embeddings from Language Models are Good Learners for Single-cell Data Analysis. Preprint, 2023. (Under review by Nature Methods)
* Liu Tianyu, Wang Yuge, Ying Rex, Zhao Hongyu<sup>* </sup>. MuSe-GNN: Learning Unified Gene Representation From Multimodal Biological Graph Data. Thirty-seventh Conference on Neural Information Processing Systems. PMLR, 2023. (poster)
* Liu Tianyu, Li Kexing, Wang Yuge, Li Hongyu, Zhao Hongyu<sup>*</sup>. Evaluating the Utilities of Foundation Models in Single-cell Data Analysis. Preprint, 2023. (Under review by Nature Methods)
* Liu Tianyu<sup>*</sup>, Greenberg Grant, Shomorony Ilan. CVQVAE: A Representation Learning based Method for Multi-omics Single Cell Data Integration. Machine Learning in Computational Biology. PMLR, 2022. (Spotlight, top 12%)
* <u>Liu Tianyu</u>, <u>Wang Yuge</u>, Zhao Hongyu<sup>*</sup>. ResPAN: a powerful batch correction model for scRNA-seq data through residual adversarial networks. Bioinformatics. 2022 Aug 10;38(16):3942-3949. doi: 10.1093/bioinformatics/btac427. PMID: 35771600; PMCID: PMC9364370.
* Liu Tianyu, Zhang Lingyu<sup>*</sup>. Apply Artificial Neural Network to Solving Manpower Scheduling Problem. 4th International Conference on Big Data and Artificial Intelligence (BDAI), 2021, pp. 58-64, doi: 10.1109/BDAI52447.2021.9515298. (Oral, [Best Paper Award](https://www.bdai.net/2021.html))

# Talk

* Second Annual Yale Single Cell Symposium
* 2nd Open Problems in Single-cell Competition during NeurIPS 2022
* Novo Nordisk, Oct 2023
* Emory University, Oct 2023
* Chan Zuckerberg Initiative, Mar 2024

# Services

As reviewers:
* Journals: PLOS Comp Bio, BMC Bioinformatics, TNNLS, TALLIP
* Conferences: NeurIPS, ML4H, MLCB, CVPR, ICLR, ICML, AAAI, AISTATS, ARR
* Competitions: 3rd Open Problems in Single-cell Competition

As organizers:
* AI+Music social activity: NeurIPS 2023.
* LLMs for biomedical research meeting: EMNLP 2024.

As contributors:
* open-source softwares including [scGLUE](https://github.com/gao-lab/GLUE/pull/114), [guidance](https://github.com/guidance-ai/guidance/pull/656), [Deeprobust](https://github.com/DSE-MSU/DeepRobust/pull/154), and [scVI](https://github.com/scverse/scvi-tools/pull/2775).


# Mentees
* Wenxin Long (BS @ NKU -> PhD @ PSU)
* Zhiyuan Cao (BS @ SJTU, MS @ Yale -> PhD @ Yale)
* Kexing Li (BS @ THU -> PhD @ Yale)
* Bizhe Bai (ML4H mentee, BS @ UoT, MS @ UQ)
* MairaÂ Elahi (ML4H mentee, Riverside Secondary School)

![Example](/images/col_institue.png)

<!--
{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}
-->
<!--
  {% include base_path %}
  {% for post in site.publications reversed %}
  { % include archive-single.html %}
  {% endfor %}
-->
